webapp/src/api.rs

```diff
@@ -93,18 +97,32 @@ pub async fn version() -> Json<Version> {
     })
 }
 
+#[utoipa::path(
+    get,
+    path = "/api/newaddress",
+    responses(
+       (status = 200, description = "Returns an unused on-chain address", body = String)
+)
+)]
 pub async fn get_unused_address() -> Result<impl IntoResponse, AppError> {
     let address = ln_dlc::get_unused_address()?;
 
     Ok(address)
 }
 
-#[derive(Serialize)]
+#[derive(Serialize, ToSchema)]
 pub struct Balance {
     on_chain: u64,
     off_chain: Option<u64>,
 }
 
+#[utoipa::path(
+get,
+path = "/api/balance",
+responses(
+(status = 200, description = "Returned on- and off-chain balance", body = Balance)
```

Comment:
```suggestion
(status = 200, description = "Returned on-chain and off-chain balance", body = Balance)
```

Reply:
Formatting is a bit weird here and elsewhere.

Reply:
Seems like it should read "Returns" not "Returned".

Reply:
I guess we should be consistent. I'm not sure what the standard is for these `description`s.

Reply:
> Formatting is a bit weird here and elsewhere.

Rust formatter doesn't work here :/

> I guess we should be consistent. I'm not sure what the standard is for these descriptions.

Not sure neither 🙈